pkg/errors/errors.go

```diff
@@ -441,6 +441,10 @@ func (e *ServiceError) IsFailedToCheckQuota() bool {
 	return e.Code == FailedToCheckQuota("").Code
 }
 
+func (e *ServiceError) IsNotFound() bool {
```

Comment:
There is a method for this already which you can reuse https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/2e1ab58534fc6319684bee2ac8ad269ea18c3eae/pkg/errors/errors.go#L377

Reply:
Kinda weirdly named, but I can re-use that.